mod/photos.php

```diff
@@ -1807,16 +1805,14 @@ function photos_content(App $a) {
 
 	$photos = array();
 	if (DBM::is_result($r)) {
-		$twist = 'rotright';
+		// "Twist" is only used for the duepunto theme with style "slackr"
```

Comment:
Shouldn’t it be a hook instead then?

Reply:
This comment is a hint that we should change this code in the future.